cpp/tests/test_abm_world.cpp

```diff
@@ -545,179 +558,3 @@ TEST(TestWorld, checkParameterConstraints)
     params.get<mio::abm::LockdownDate>() = mio::abm::TimePoint(-2);
     ASSERT_EQ(params.check_constraints(), true);
 }
-
-TEST(TestWorld, copyWorld)
```

Comment:
Why was this removed? Because we use default (or almost default) copy constructor?

Reply:
Exactly. I think we can trust the copy ctors. But if you do want to keep that test, we should wait for the world serialization s.t. we do not have to pseudo-implement "==" in the test.